CompositeUi/src/views/component/CardExtended.js

```diff
@@ -33,6 +34,10 @@ class CardExtended extends React.Component {
     return '';
   }
 
+  triggerOnMemberRemoveClicked(participant) {
```

Comment:
This logic is highly coupled to the memberCard, should no be inside the agnostic cardExtended component